### PR TITLE
Make grammatical improvements to print msgs

### PR DESCRIPTION
**Pull request by Milind220**
I realised that there were some print messages in the installer that were grammatically incorrect, or could just be improved. I corrected the tense of two such messages, and rephrased the third one. Pretty simple small edits, for my first PR!

### CODE REVIEW

File: installer/steps/b_pip.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import unix_windows
 
 
-section("Install requirements")
```
Comment (Milind220) [inline installer/steps/b_pip.py:5]: Ideally, this sentence should be in the present tense

File: installer/steps/c_nltk.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 PACKAGES = ["wordnet", "punkt"]
 
-section("Download additional data (Dictionary)")
```
Comment (Milind220) [inline installer/steps/c_nltk.py:7]: Same as above.

File: installer/steps/d_optional.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ def check_optional_requirement(requirement):
             package_manager = optional.get_guess(optional.PackageManager)
             cmd = "{} {}".format(package_manager, package)
 
-            print("\nOur Guess how to install:\n>{}".format(cmd))
+            print("\nOur tip on how to install this: \n>{}".format(cmd))
```
Comment (Milind220) [inline installer/steps/d_optional.py:63]: This was grammatically incorrect/badly phrased earlier. Just rephrased it :)

**Comment by pnhofmann** [comment]
Thank you ;)

**Pull request merged**